test/clj/rems/test_browser.clj

```diff
@@ -72,6 +72,24 @@
   (btu/wait-page-loaded)
   (btu/screenshot (io/file btu/reporting-dir "administration-resources-page.png")))
 
+(defn go-to-admin-forms []
+  (click-administration-menu "Forms")
+  (btu/wait-visible {:tag :h1 :fn/text "Forms"})
+  (btu/wait-page-loaded)
+  (btu/screenshot (io/file btu/reporting-dir "administration-forms-page.png")))
+
+(defn go-to-admin-workflows []
+  (click-administration-menu "Workflows")
+  (btu/wait-visible {:tag :h1 :fn/text "Workflows"})
+  (btu/wait-page-loaded)
+  (btu/screenshot (io/file btu/reporting-dir "administration-workflows-page.png")))
+
+(defn go-to-admin-catalogue []
```

Comment:
Perhaps we could use a slightly more generic helper that would be like `(go-to-admin "Workflows")` that would screenshot `administration-workflows-page.png`?

Reply:
done